iterator.go

```diff
@@ -169,6 +169,8 @@ func (item *Item) yieldItemValue() ([]byte, func(), error) {
 		vp.Decode(item.vptr)
 		result, cb, err := item.db.vlog.Read(vp, item.slice)
 		if err != ErrRetry {
+			//item.db.opt.Logger.Errorf("Unable to read: Key: %v, Version : %v, meta: %v userMeta: %v",
```

Comment:
remove dead code.

Reply:
Thanks, but I was testing this on TeamCity. This needs to go in.